### PR TITLE
Feature stud 773 repo migration

### PR DESCRIPTION
**Pull request by joecotton-wk**
Updating Golang package name

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: skynet.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Testing with Skynet - https://github.com/Workiva/skynet/tree/master/docs
+name: xlsx-unit-tests
+image: "golang:bullseye"
+description: run xlsx unit tests
+size: small
+contact: support-onecloud
+
+run:
+  on-pull-request: yes
```
Comment (micahbeeman-wf) [inline skynet.yaml]: This should be true instead of yes

File: skynet.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  on-tag: yes
```
Comment (micahbeeman-wf) [inline skynet.yaml]: This should be true instead of yes

File: skynet.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Testing with Skynet - https://github.com/Workiva/skynet/tree/master/docs
+name: xlsx-unit-tests
+image: "golang:bullseye"
```
Comment (micahbeeman-wf) [inline skynet.yaml]: No need for the quotes around the image

**Comment by joecotton-wk** [comment]
grg

**Review by danerubado-wk — APPROVED** [review]
lgtm

**Comment by danerubado-wk** [comment]
QA+1

**Comment by joecotton-wk** [comment]
@Workiva/release-management-p

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**